include/sharg/detail/type_name_as_string.hpp

```diff
@@ -49,21 +49,26 @@ inline std::string const type_name_as_string = [] ()
     // https://gcc.gnu.org/onlinedocs/libstdc++/libstdc++-html-USERS-4.3/a01696.html
     int status{};
     safe_ptr_t demangled_name_ptr{abi::__cxa_demangle(typeid(type).name(), 0, 0, &status),
-                                  [] (char * name_ptr) { free(name_ptr); }};
+                                  [](char * name_ptr)
+                                  {
+                                      free(name_ptr);
+                                  }};
 
     // We exclude status != 0, because this code can't be reached normally, only if there is a defect in the compiler
     // itself, since the type is directly given by the compiler. See https://github.com/seqan/seqan3/pull/2311.
     // LCOV_EXCL_START
+    // clang-format off
     if (status != 0)
         return std::string{typeid(type).name()} +
                " (abi::__cxa_demangle error status (" + std::to_string(status) + "): " +
                (status == -1 ? "A memory allocation failure occurred." :
-               (status == -2 ? "mangled_name is not a valid name under the C++ ABI mangling rules." :
-               (status == -3 ? "One of the arguments is invalid." : "Unknown Error"))) + ")";
+                   (status == -2 ? "mangled_name is not a valid name under the C++ ABI mangling rules." :
+                       (status == -3 ? "One of the arguments is invalid." : "Unknown Error"))) + ")";
```

Comment:
did you forget to revert the change after turning off clang format?

Reply:
I wanted to emphasize the if/else nature of this block a bit more, but I can also revert it?